bazel/cas/service.go

```diff
@@ -154,15 +181,17 @@ func (s *casServer) Read(req *bytestream.ReadRequest, ser bytestream.ByteStream_
 		log.Infof("Opening store resource for reading: %s", storeName)
 		r, err = s.storeConfig.Store.OpenForRead(storeName)
 		if err != nil {
-			log.Errorf("Failed to OpenForRead: %v", err)
-			return status.Error(codes.NotFound, fmt.Sprintf("Failed opening resource %s for read, returning NotFound. Err: %v", storeName, err))
+			// Reset err to nil - don't count NotFound towards request failures
+			openErr := err
```

Comment:
this reads a little weird... not sure how to clean it up though. maybe just a little more context in the comment as to why this isn't counted as a request failure.